tests/test_schema.py

```diff
@@ -1159,3 +1159,29 @@ class Ex2(msgspec.Struct):
             "required": ["b"],
         },
     }
+
+
+def test_multiline_docstrig():
+    class Example(msgspec.Struct):
+        """
+        first line
+
+            indented
+
+        last line.
+        """
+
+        pass
+
+    assert msgspec.json.schema(Example) == {
+        "$ref": "#/$defs/Example",
+        "$defs": {
+            "Example": {
+                "description": "\nfirst line\n\n    indented\n\nlast line.\n",
```

Comment:
If we do want to strip leading/trailing whitespace (see above), we'll need to update this test too:

```suggestion
                "description": "first line\n\n    indented\n\nlast line.",
```